redaxo/src/core/lib/sql/sql.php

```diff
@@ -755,7 +768,11 @@ protected function fetchValue($feldname)
     public function getRow($fetch_type = PDO::FETCH_ASSOC)
     {
         if (!$this->lastRow) {
-            $this->lastRow = $this->stmt->fetch($fetch_type);
+            $lastRow = $this->stmt->fetch($fetch_type);
+            if (false === $lastRow) {
+                throw new rex_sql_exception('unable to fetch');
```

Comment:
https://github.com/redaxo/redaxo/issues/3730

Da bekam man in 5.10 als Exception 
> Field "template.name" does not exist in result!

Jetzt bekommt man nur noch
> unable to fetch

Das ist noch nicht so optimal.

Reply:
Gibt es infos die man in der exception noch ergänzen sollte für einfacheres debugging?

Reply:
Mein erster Eindruck war, dass die neue Exception ungenauer wäre. Aber nachdem ich gerade nochmal drüber nachgedacht habe, ist sie ja eigentlich doch spezifischer, da in dem Fall nun mal die ganze Row nicht gefetcht werden konnte, und nicht nur eine Spalte fehlt.

Reply:
Ggf drauf hinweisen dass man ggf checken muss ob rows getroffen wurden?